text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Fabric_Website
+nav_order: 3
+---
+
+- Feature Name: Fabric Website
+- Start Date: (fill me in with today's  date, 2020-08-14)
+- RFC PR: (leave this empty)
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC details the proposal for a new website for using Hyperledger Fabric. This website would serve as a "front door" for the project, and collate the various resources available to help users get started with Fabric.
+
+The website has been deployed to Heroku for development purposes and is available to browse [here](https://warm-reaches-04614.herokuapp.com/).
+
+# Motivation
+[motivation]: #motivation
+
+Presently, there are multiple different places to get started with Hyperledger Fabric. It can be difficult for a new user to know which resource is the best one for them to start with based on what they would like to achieve. The available documentation has excellent content for getting started and explains everything well, but there is also a large amount of additional content on other relevant topics (for example, performance testing) that is also useful to new users.
+
+This website aims to surface this content and bring it with the existing documentation into one easy to consume place for users.
```

Comment:
Regarding the getting started under the developer page, would it make more sense to link to the wiki page for installing necessary development tools/prerequisites instead of linking to the IBM VS Code extension? Having a link to the extension is probably fine, but not everyone will intend on using the VS Code extension for setting up their development environment.